src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -73,127 +74,127 @@
-        notificationRequest.setNotificationEmail(Objects.toString(callbackRequest.getCaseDetails().getData().get("generalEmailRecipient")));
-        sendNotificationEmail(notificationRequest, uri);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        applicantNotificationRequest.setNotificationEmail(Objects.toString(callbackRequest.getCaseDetails().getData().get(GENERAL_EMAIL_RECIPIENT)));
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     public void sendContestOrderNotApprovedEmail(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedOrderNotApproved());
-        notificationRequest = createNotificationRequest(callbackRequest);
-        sendNotificationEmail(notificationRequest, uri);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     public void sendContestedConsentOrderApprovedEmail(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedConsentOrderApproved());
-        notificationRequest = createNotificationRequest(callbackRequest);
-        sendNotificationEmail(notificationRequest, uri);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     public void sendContestedConsentOrderNotApprovedEmail(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedConsentOrderNotApproved());
-        notificationRequest = createNotificationRequest(callbackRequest);
-        sendNotificationEmail(notificationRequest, uri);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     public void sendContestedConsentGeneralOrderEmail(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedConsentGeneralOrder());
-        notificationRequest = createNotificationRequest(callbackRequest);
-        sendNotificationEmail(notificationRequest, uri);
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
+        sendNotificationEmail(applicantNotificationRequest, uri);
     }
 
     public void sendContestedGeneralApplicationReferToJudgeEmail(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedGeneralApplicationReferToJudge());
-        notificationRequest = createNotificationRequest(callbackRequest);
-        notificationRequest.setNotificationEmail(Objects.toString(
+        applicantNotificationRequest = createNotificationRequestForAppSolicitor(callbackRequest);
```

Comment:
I guess this is for judge Email not for applicant solicitor

Reply:
Good spot - renamed this to reflect that